HoloRepositoryDemoApplication/Assets/Scripts/HologramDisplayScene.cs

```diff
@@ -0,0 +1,39 @@
+﻿using UnityEngine;
+using HoloStorageConnector;
+using TMPro;
+
+public class HologramDisplayScene : MonoBehaviour
+{   
+    [SerializeField]
+    private TextMeshProUGUI PatientName = null;
+    [SerializeField]
+    private TextMeshProUGUI PatientInfo = null;
+    [SerializeField]
+    private TextMeshProUGUI HologramTitle= null;
+    [SerializeField]
+    private TextMeshProUGUI HologramInfo = null;
+
+    public static Patient Patient;
+    public static Hologram Hologram;
+
+    void Start()
+    {
+        PatientName.text = $"{Patient.name.given} {Patient.name.family}";
```

Comment:
The server does not return `name.full`, so it will be null, so i changed the defination when mapp the json data, i will change it back when the server return `name.full`
![image](https://user-images.githubusercontent.com/46536567/62795978-e84f0480-bacf-11e9-8bd6-83fd786537d2.png)

Reply:
Oh okay. Good idea!